src/core/quantum_gate.jl

```diff
@@ -1506,12 +1506,11 @@ Return the `Operator` which applies a ``π/2`` rotation about the ``Z`` axis.
 The `Operator` is defined as:
```

Comment:
also on line 1518

Reply:
I understand what you mean by "phase offset is symmetric", but I can't find it by googling. I've added that the unitary is symmetric, but "symmetric phase offset" sounds strange to me. Do you have a reference of it somewhere?

Reply:
No, I came up with that, which isn't really standard. Also, saying "the unitary is symmetric" is misleading as "symmetric" for matrices means `U^T = U`. We could put it as "the phase offset is applied symmetrically to the |0> and |1> states". Either that or remove it altogether. Qiskit doesnt make any mention of it, just gives the matrices.

Reply:
While the rotation gates are symmetric, that probably isn't useful information. I removed the mention of symmetry since we aren't really going into the properties of other operators.